parameters/world_modeling/world_model_demo_intern.yaml

```diff
@@ -8,15 +7,16 @@ plugins:
 #- name: robot_plugin
 #  lib: libed_robot_plugin.so
 #  parameters:
-#      robot_name: amigo
-#      urdf_rosparam: /amigo/robot_description
+#      robot_name: hero
```

Comment:
Why is this plugin disabled?

Reply:
I have been focused on the free mode, so this one is still WIP